mindmap/mindmap.py

```diff
@@ -93,20 +108,81 @@ def studio_view(self, context=None):
         frag.initialize_js("MindMapXBlock")
         return frag
 
-    # TO-DO: change this handler to perform your own actions.  You may need more
-    # than one handler, or you may not need any handlers at all.
-    @XBlock.json_handler
-    def increment_count(self, data, suffix=''):
+    def connect_to_s3(self):
+        """
+        Create a connection to S3.
+
+        Returns:
+            boto3.client: The connection to S3.
         """
-        An example handler, which increments the data.
+        AWS_ACCESS_KEY_ID = getattr(settings, "AWS_ACCESS_KEY_ID", None)
+        AWS_SECRET_ACCESS_KEY_ID = getattr(settings, "AWS_SECRET_ACCESS_KEY_ID", None)
```

Comment:
```suggestion
        aws_access_key_id = getattr(settings, "AWS_ACCESS_KEY_ID", None)
        aws_secret_access_key_id = getattr(settings, "AWS_SECRET_ACCESS_KEY_ID", None)
```
We try using capital letter constants when we define them on top of the file, but here we can use them like that. Question: what should happen if they're not defined?

Reply:
It should raise an exception. I will add it.